### DISCUSSION TITLE
System prompts: Can I define a default to always be used? What is the built-in one? Do our custom ones overwrite or extend the built-in prompt?

### DISCUSSION THREAD

**Discussion by ericmburgess** [brianpetro/obsidian-smart-connections#717] [Q&A]
Tried to put it all in the title for clarity, but these are my questions around system prompts:

1. Can I define a default to always be used? The vast majority of my queries are similar and should use the same system prompt, and only rarely would I choose another more specialized one. It's kind of tedious if I need to select my `default` every time. Speaking of which, I created that, hoping it would be used as the default, but it's not.
2. What is the built-in system prompt? Knowing that would be very helpful in crafting my own. 
3. Do our custom ones replace or extend the built-in prompt, or are they incorporated in some different way?

**Comment by brianpetro** [comment]
Hi @ericmburgess 

1) this is something that will be enabled in an upcoming version 
2) it's very minimal and I believe it's only used when notes are being used as context, something like "use the following as context and begin response with 'Based on your notes...'"
3) extend

I hope that helps. A lot of the above will be clarified with upcoming updates that will focus on improving the Smart Chat. I'll be sharing more of those details soon
🌴
